.rubocop.yml

```diff
@@ -9,7 +9,6 @@ AllCops:
   DisplayStyleGuide: true
   ExtraDetails: true
   UseCache: true
```

Comment:
Other thing to undo
```suggestion
  UseCache: true
  CacheRootDirectory: tmp
```

Reply:
https://github.com/mastodon/mastodon/pull/22952/commits/b4a37cf005cf0e8f8f093f9b447ff9c82ba5b0de

Reply:
> Can you rebase against main? Also I would not include CacheRootDirectory or ruby version changes in this PR as it is unrelated. Otherwise, LGTM.

Reply:
I did, 10 minutes ago

Reply:
ah, I didn't parse that in Clairs comment. Sorry.

Reply:
removed the commit then.